src/core/utils/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { getWindow } from '../functions';
+
+export const useMediaQuery = (queryString: string) => {
+  const [matches, setMatches] = React.useState<boolean>();
+
+  React.useLayoutEffect(() => {
+    const mediaQueryList = getWindow()?.matchMedia?.(queryString);
+    const handleChange = ({ matches }: MediaQueryListEvent) =>
+      setMatches(matches);
+
+    if (mediaQueryList != undefined) {
+      setMatches(mediaQueryList.matches);
+      try {
+        mediaQueryList.addEventListener('change', handleChange);
+      } catch (_) {
+        // Safari 13 fallback
+        mediaQueryList.addListener?.(handleChange);
+      }
+    }
+
+    return () => {
+      try {
+        mediaQueryList?.removeEventListener('change', handleChange);
+      } catch (_) {
```

Comment:
brackets should not be needed if you dont need error object

Reply:
Removed